actions/setup-cf-cli/action.yml

```diff
@@ -0,0 +1,48 @@
+name: Setup the Cloud Foundry CLI
+description: Step to install the cloud foundry cli on the agent.
+inputs:
+  CF_USERNAME:
+    description: Username to login to the PaaS space
+    required: true
+  CF_PASSWORD:
+    description: Password of the user to login to the PaaS space
+    required: true
+  CF_SPACE_NAME:
+    description: Name of the PaaS space to set the target context
+    required: true
+  CF_ORG_NAME:
+    description: Name of the PaaS organisation 
+    required: false
+    default: dfe-teacher-services
+  CF_API_URL:
+    description: PaaS API endpoint, default https://api.london.cloud.service.gov.uk
+    required: false
+    default: https://api.london.cloud.service.gov.uk
+  INSTALL_CONDUIT:
+    description: Set to true to install the cf conduit extension
+    required: false
+    default: 'false'
+runs:
+  using: composite
+  steps: 
+    - name : Install cf client
+      shell: bash
+      run: |
+        curl -sL ${CF_CLI_DOWNLOAD_URL} | sudo tar -zx -C /usr/local/bin
+        sudo mv /usr/local/bin/cf7 /usr/local/bin/cf 
+        if [ "${{ inputs.INSTALL_CONDUIT }}" == "true" ]
+        then
+          cf install-plugin conduit -f
+        fi
+      env:
+        CF_CLI_DOWNLOAD_URL: 'https://packages.cloudfoundry.org/stable?release=linux64-binary&version=7.1.0'
```

Comment:
Can we move this to the top for more visibility?

Reply:
I kind of the like to keep the `env:` element after the script, reads nice for me and keeps the script between elements.
can change if you feel otherwise

Reply:
have moved it to the top :)